picam.py

```diff
@@ -0,0 +1,36 @@
+"""
+Taking images with a Raspberry pi camera module 3 on a Rpi on the drone.
+DO NOT ACTIVATE VENV WHEN USING THIS SCRIPT, IT IS MEANT TO BE USED WITH THE DEFAULT
+RASPBERRY PI LIBRARIES THAT ARE PRE-INSTALLED. Those libraries cannot be easily
+installed using pip nad requires additionaly steps, so we just use the provided
+library in global space.
```

Comment:
Replace `Those libraries ...` with `See documentation for more detail.` and add this explanation to Confluence.